datatypes/papas/Definitions.h

```diff
@@ -0,0 +1,25 @@
+//
+//  Definitions.h
+//  papas
+//
+//  Created by Alice Robson on 12/05/16.
+//
+//
+
+#ifndef Definitions_h
+#define Definitions_h
+
+#include <inttypes.h>
+
+namespace papas {
+typedef uint64_t IdType;
+
+/// describe position of a cluster within the Detector
+enum Position { kVertex, kEcalIn, kEcalOut, kEcalDecay, kHcalIn, kHcalOut };
```

Comment:
I'm wondering whether we should have these enums floating around in the papas namespace. Couldn't we put them e.g. in the detector class?